CHANGELOG.md

```diff
@@ -8,6 +8,9 @@
 * Fixed the GitHub url used by Help -> Submit a ticket.
 * Removed several unused dependencies, including JOGL, JTS, OpenCSV, Trove4j, JScience, and XML-APIs.
 * Updated several dependencies to the latest versions, including Geotools, Jetty, Apache Commons, Jackson, RSyntaxArea, Google Guava, Apache POI, EJML, Processing, Jython, and SwingX.
+* Updated ConstellationLogger new methods viewStarted, viewStopped and viewInfo to support logging of Views.
```

Comment:
A minor comment but should this read "Updated ConstellationLogger *with* new methods..." ?

Reply:
Yep - fixed.